app/controllers/cooks_controller.rb

```diff
@@ -0,0 +1,74 @@
+class CooksController < ApplicationController
+  before_action :set_cook, only: [:show, :edit, :update, :destroy]
+
+  # GET /cooks
+  # GET /cooks.json
+  def index
+    @cooks = Cook.all
+  end
+
+  # GET /cooks/1
+  # GET /cooks/1.json
+  def show
+  end
+
+  # GET /cooks/new
+  def new
+    @cook = Cook.new
+  end
+
+  # GET /cooks/1/edit
+  def edit
+  end
+
+  # COOK /cooks
+  # COOK /cooks.json
+  def create
+    @cook = Cook.new(cook_params)
+
+    respond_to do |format|
+      if @cook.save
+        format.html { redirect_to @cook, notice: 'Cook post was successfully created.' }
+        format.json { render :show, status: :created, location: @cook }
+      else
+        format.html { render :new }
+        format.json { render json: @cook.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /cooks/1
+  # PATCH/PUT /cooks/1.json
+  def update
+    respond_to do |format|
+      if @cook.update(post_params)
+        format.html { redirect_to @cook, notice: 'Cook post was successfully updated.' }
+        format.json { render :show, status: :ok, location: @cook }
+      else
+        format.html { render :edit }
+        format.json { render json: @cook.errors, status: :unprocessable_entity }
+      end
+    end
+  end
```

Comment:
Do you need the ability to update cooks yet?

Reply:
No, not according to my tests! I've deleted this now.